notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb

```diff
@@ -4,31 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>The <code>else: continue</code> statement is redundant here. You can safely remove the <code>else</code> and <code>continue</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>